internal/utils/replaceStrings.go

```diff
@@ -16,8 +16,11 @@ const (
 
 var kernelRegexp = regexp.MustCompile("[.,-]")
 
-func KernelComponentsAsEnvVars(kernel string) []string {
+func KernelComponentsAsEnvVars(kernel string) ([]string, error) {
 	osConfigFieldsList := kernelRegexp.Split(kernel, -1)
+	if len(osConfigFieldsList) < 3 {
+		return nil, fmt.Errorf("invalid kernel version")
```

Comment:
Can we return a more detailed error message? For example, "expecting the %s format but got %s" or something similar.